lib/generate-svg.js

```diff
@@ -40,7 +40,8 @@ module.exports = (sources = [], options = {}, warnings = []) => {
                 title: true,
                 symbol: true,
                 use: false,
-                view: false
+                view: false,
+                keepDimensions: false
```

Comment:
I'd be in favor of just calling this `dimensions` to align it with the naming of the other options.

Reply:
Changed naming [85c8b32](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/pull/219/commits/85c8b328abd9769543e0cee2453dc9c3ed06d1f4)